python/server.py

```diff
@@ -66,256 +107,371 @@ def empty_to_none(field):
 
 item_id = None
 
+
 @app.route('/api/info', methods=['POST'])
 def info():
-  global access_token
-  global item_id
-  return jsonify({
-    'item_id': item_id,
-    'access_token': access_token,
-    'products': PLAID_PRODUCTS
-  })
+    global access_token
+    global item_id
+    return jsonify({
+        'item_id': item_id,
+        'access_token': access_token,
+        'products': PLAID_PRODUCTS
+    })
+
 
 @app.route('/api/create_link_token_for_payment', methods=['POST'])
 def create_link_token_for_payment():
-  global payment_id
-  try:
-    create_recipient_response = client.PaymentInitiation.create_recipient(
-      'Harry Potter',
-      'GB33BUKB20201555555555',
-      {
-        'street':      ['4 Privet Drive'],
-        'city':        'Little Whinging',
-        'postal_code': '11111',
-        'country':     'GB'
-      },
-    )
-    recipient_id = create_recipient_response['recipient_id']
-
-    create_payment_response = client.PaymentInitiation.create_payment(
-      recipient_id,
-      'paymentRef',
-      {
-        'value': 12.34,
-        'currency': 'GBP'
-      },
-    )
-    pretty_print_response(create_payment_response)
-    payment_id = create_payment_response['payment_id']
-    response = client.LinkToken.create(
-      {
-        'user': {
-          # This should correspond to a unique id for the current user.
-          'client_user_id': 'user-id',
-        },
-        'client_name': "Plaid Quickstart",
-        'products': PLAID_PRODUCTS,
-        'country_codes': PLAID_COUNTRY_CODES,
-        'language': "en",
-        'redirect_uri': PLAID_REDIRECT_URI,
-        'payment_initiation': {
-          'payment_id': payment_id
-        }
-      }
-    )
-    pretty_print_response(response)
-    return jsonify(response)
-  except plaid.errors.PlaidError as e:
-    return jsonify(format_error(e))
+    global payment_id
+    try:
+        rec_create_request = PaymentInitiationRecipientCreateRequest(
+            name='John Doe',
+            bacs=NullableRecipientBACS(account='26207729', sort_code='560029'),
+            address=PaymentInitiationAddress(
+                street=['street name 999'],
+                city='city',
+                postal_code='99999',
+                country='GB'
+            )
+        )
+        create_recipient_response = client.payment_initiation_recipient_create(
+            rec_create_request)
+        recipient_id = create_recipient_response['recipient_id']
+
+        payment_create_request = PaymentInitiationPaymentCreateRequest(
+            recipient_id=recipient_id,
+            reference='TestPayment',
+            amount=Amount(
+                currency='GBP',
+                value=100.00
+            )
+        )
+        create_payment_response = client.payment_initiation_payment_create(
+            payment_create_request
+        )
+        print(create_payment_response)
```

Comment:
lets remove all stray `print` statements and just `pretty_print_response` in the same places the old code did

Reply:
so the pretty_print_response does not work with the new clibs, and print looks fine in the console.

Reply:
so the pretty_print_response does not work with the new clibs.  print looks fine.